ui/src/main/java/com/hyperwallet/android/ui/repository/UserRepositoryImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.hyperwallet.android.ui.repository;
+
+import android.os.Handler;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+
+import com.hyperwallet.android.Hyperwallet;
+import com.hyperwallet.android.exception.HyperwalletException;
+import com.hyperwallet.android.listener.HyperwalletListener;
+import com.hyperwallet.android.model.HyperwalletUser;
+
+public class UserRepositoryImpl implements UserRepository {
```

Comment:
please add a method called void refreshUser() which sets the mUser variable to null. This can be useful if the user profile is updated